CONTRIBUTING.md

```diff
@@ -64,6 +64,18 @@ Running the code
 
 Note: The tests are headless, you won't see any UI. You can use `print` to generate console output or you can interact with the DartVM via observatory at [http://localhost:8181/](http://localhost:8181/).
 
+Running the examples
+--------------------
+
+* You can find example code in subdirectories of the `examples` directory, for example `examples/stocks`.
+* Once you have a local build, run `pub get` from the example folder of your choice to make sure that you have all of the Dart dependencies.
+* Then, to run the current example locally, you can run:
+ - `$ ./packages/sky/sky_tool start --build-path ../../out/Debug/`
+* The `--build-path` parameter is the path to your build directory for the build you want to run.  To run on a connected Android device, you could set it to `../../out/android_Debug/`, assuming your output directory is set up normally.
```

Comment:
This makes me wonder if we should have a --local-build=Debug option instead which was smarter.